hack/make-rules/local-up.sh

```diff
@@ -153,9 +153,10 @@ function print_success() {
   echo -e "  export KUBECONFIG=${HOST_CLUSTER_KUBECONFIG}"
   echo -e "\nTo observe the status of KubeAdmiral control-plane components, run:"
   echo -e "  export KUBECONFIG=${META_CLUSTER_KUBECONFIG}"
-  echo -e "\nTo manage your member clusters, run:"
-  echo -e "  export KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG}"
-  echo "Please use 'kubectl config use-context ${MEMBER_CLUSTER_NAME_PREFIX}-1/${MEMBER_CLUSTER_NAME_PREFIX}-2/${MEMBER_CLUSTER_NAME_PREFIX}-3' to switch to the different member cluster."
+  echo -e "\nTo manage your member clusters, run one of the following:"
```

Comment:
```suggestion
  echo -e "\nTo inspect your member clusters, run one of the following:"
```
`manage` suggests cluster lifecycle operations, e.g., creating / destroying / joining / removing clusters.

Reply:
The content has been changed.